server/http_status.go

```diff
@@ -151,6 +151,8 @@ func (s *Server) startHTTPServer() {
 	router.Handle("/mvcc/hex/{hexKey}", mvccTxnHandler{tikvHandlerTool, opMvccGetByHex})
 	router.Handle("/mvcc/index/{db}/{table}/{index}/{handle}", mvccTxnHandler{tikvHandlerTool, opMvccGetByIdx})
 
+	// HTTP path for generate metric profile.
+	router.Handle("/metric/profile", profileHandler{tikvHandlerTool})
```

Comment:
em, we have already provided a `/metrics` path, so here providing another `/metric` seems a little strange.